easybuild/tools/options.py

```diff
@@ -357,6 +357,9 @@ def override_options(self):
             'consider-archived-easyconfigs': ("Also consider archived easyconfigs", None, 'store_true', False),
             'containerize': ("Generate container recipe/image", None, 'store_true', False, 'C'),
             'copy-ec': ("Copy specified easyconfig(s) to specified location", None, 'store_true', False),
+            'cuda-cache-maxsize': ("Maximum size of the CUDA cache (in bytes) used for JIT compilation of PTX code. "
```

Comment:
Would a size in MB make more sense?

I'd say it's very unlikely that people would want to use a cache size smaller than 1MB?

Reply:
Ok done, changed this to be MiB